<launch>

    <param name="apriltags2_ros/optical_flow_accelerated" type="bool" value="true"/>

    <include file="$(find opencv_apps)/launch/lk_flow.launch">
        <arg name="image" default="camera/color/image_raw" doc="The image topic. Should be remapped to the name of the real image topic." />
        <arg name="use_camera_info" default="true" doc="Indicates that the camera_info topic should be subscribed to to get the default input_frame_id. Otherwise the frame from the image message will be used." />
        <arg name="debug_view" default="false" doc="Specify whether the node displays a window to show edge image" />
    </include>
    <include file="$(find apriltags2_ros)/launch/continuous_pose_detection.launch" />
</launch>
